src/ReactiveSocket.cpp

```diff
@@ -75,6 +76,20 @@ void ReactiveSocket::requestSubscription(
   automaton->start();
 }
 
+void ReactiveSocket::requestFireAndForget(
+    Payload request,
+    Subscriber<Payload>& responseSink) {
```

Comment:
I'm going to get rid of it as the Subscriber here would be very superficial. But let me know if you think it is needed.